contrib/devtools/circular-dependencies.py

```
@@ -34,13 +34,13 @@ def module_name(path):
     return None
 
 files = dict()
-deps: Dict[str, Set[str]] = dict()
+deps: dict[str, Set[str]] = dict()
```

Flake8 (7.3.0)

[error] undefined name 'Set'

(F821)